src/main/java/org/mongeez/MongeezRunner.java

```diff
@@ -50,23 +50,23 @@ public void execute() {
         Mongeez mongeez = new Mongeez();
         mongeez.setMongo(mongo);
         mongeez.setDbName(dbName);
-        
+
         if(changeSetsValidator != null) {
             mongeez.setChangeSetsValidator(changeSetsValidator);
         }
         else {
             mongeez.setChangeSetsValidator(new DefaultChangeSetsValidator());
         }
-        
+
         if (changeSetFileProvider != null) {
             mongeez.setChangeSetFileProvider(changeSetFileProvider);
         } else {
             mongeez.setFile(file);
+        }
 
-            if(!StringUtils.isEmpty(userName) && !StringUtils.isEmpty(passWord)){
-            	MongoAuth auth = new MongoAuth(userName, passWord, authDb);
-                mongeez.setAuth(auth);
-            }
+        if(StringUtils.isNotEmpty(userName) && StringUtils.isNotEmpty(passWord) && StringUtils.isNotEmpty(authDb)){
```

Comment:
Was the fix just the isEmpty check? Can you elaborate what the fix was?

Reply:
Earlier the `auth` was set only when `changeSetFileProvider` was `null`. https://github.com/causecode/mongeez/pull/1/files/4ae9bcaa9b1b07d8d41d30623be37ae198df90f9#diff-51e9892981813134d954c8a18cd8b444R61

I moved the authentication part outside of `else` block so that when `username, password and authDb` is present, `auth` field should be set, irrespective of the value of `changeSetFileProvider`.